pkg/build/builder/daemonless.go

```diff
@@ -61,23 +61,7 @@ func pullDaemonlessImage(sc types.SystemContext, store storage.Store, imageName
 	// if credsDir, ok := os.LookupEnv("PULL_DOCKERCFG_PATH"); ok {
 	// 	systemContext.AuthFilePath = filepath.Join(credsDir, "config.json")
 	// }
-	systemContext.AuthFilePath = "/tmp/config.json"
```

Comment:
(Maybe delete the comment above as well, in all of the modified functions.)

Reply:
part of next push